src/components/POLine/POLineForm.js

```diff
@@ -251,13 +257,17 @@ class POLineForm extends Component {
                         change={change}
                         dispatch={dispatch}
                         formValues={formValues}
+                        currencies={vendorCurrencies}
                       />
                     </Accordion>
                     <Accordion
                       label={<FormattedMessage id="ui-orders.line.accordion.vendor" />}
                       id={ACCORDION_ID.vendor}
                     >
-                      <VendorForm {...this.props} />
+                      <VendorForm
+                        accounts={accounts}
+                        {...this.props}
```

Comment:
Are you sure you need all the props down there? Could you please check what are required and pass only them? thanks

Reply:
<VendorForm> contains from redux-form <Field> and I'm not sure we need to pass separately all required props for them